procedures: Allow to disable 'Install from VSIX' command using ConfigMap (#2890)
codenvy/che-docs: modules/administration-guide/pages/editor-configurations-for-microsoft-visual-studio-code.adoc

* settings.json
* extensions.json
* product.json
* configurations.json

The *settings.json* section contains various settings with which you can customize different parts of the Code - OSS editor. +
The *extensions.json* section contains recommended extensions that are installed when a workspace is started. +
The *product.json* section contains properties that you need to add to the editor's *product.json* file. If the property already exists, its value will be updated. +

> The *configurations.json* section contains properties that allows to configure Code - OSS editor. For example, `extensions.install-from-vsix-enabled` property can be used to disable `Install from VSIX` command.

Reviewer: ```suggestion
The *configurations.json* section contains properties for Code - OSS editor configuration. For example, you can use the `extensions.install-from-vsix-enabled` property to disable `Install from VSIX` command.
```

.Procedure

> * Add a new ConfigMap to the user's {orch-namespace}, define the supported sections, specify the properties you want to add.

Reviewer: ```suggestion
* Add a new ConfigMap to the user's {orch-namespace}, define the supported sections, and specify the properties you want to add.
```

Make sure that the Configmap contains data in a valid JSON format.
====

> TIP: Consider adding the ConfigMap to the `eclipse-che` namespace. It allows to replicate the ConfigMap across all user namespaces while preventing modifications within user's namespaces. See xref:configuring-a-user-namespace.adoc[].

Reviewer: ```suggestion
TIP: Consider adding the ConfigMap to the `eclipse-che` namespace. By adding it to the namespace, you replicate the ConfigMap across all user namespaces while preventing modifications within user's namespaces. See xref:configuring-a-user-namespace.adoc[].
```

> TIP: Consider adding the ConfigMap to the `eclipse-che` namespace. It allows to replicate the ConfigMap across all user namespaces while preventing modifications within user's namespaces. See xref:configuring-a-user-namespace.adoc[].

Reviewer 2: `eclipse-che` is the upstream specific, we need to make sure that this part of the docs is properly downstreamed

Author: good catch!
thank you!

I replaced it by the `{prod-namespace}`

Author: if it looks good - will prepare a PR to fix this place in the main branch
